app/javascript/src/new_voucher_form.js

```diff
@@ -1,11 +1,13 @@
 function setup() {
   if ($('#new_voucher').length > 0) {
     $('#voucher_custom_value').on('input', (ev) => {
+      $('#voucher_custom_value').parent().addClass("selected"); // I tried $(this), but didn't work
```

Comment:
@ricardogo O que é que estavas a tentar fazer, mesmo?

Reply:
está feito, mas não é eficiente. para não estar a fazer uma query nova, queria usar o $(this), que costuma funcionar, mas como não consegui, voltei a fazer o query com o ID